packages/common/src/components/Footer/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+import MaxWidth from '../flex/MaxWidth';
+import LinkGroup from './LinkGroup';
+
+import linkGroups from './linkGroups.json';
+
+const Container = styled.div`
```

Comment:
These components should go in a `elements.ts` file next to this file.
You can create a folder and make this file the `index.tsx`.